ember/ember-cli-build.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(defaults) {
       extensions: ['css', 'js'],
       exclude: ['cesium'],
     },
+    minifyJS: {
+      enabled: EmberApp.env() !== 'production',
```

Comment:
okay, thanks for looking into this! I'll merge this as is for now and will follow this up later in Ember CLI.